docs/reference/operator_parameters.md

```diff
@@ -80,7 +80,10 @@ Those are top-level keys, containing both leaf keys and groups.
   are applied. The default is `-1`.
 
 * **resync_period**
-  period between consecutive sync requests. The default is `5m`.
+  period between consecutive sync requests. The default is `30m`.
+
+* **repair_period**
+  period between consecutive repair requests. The default is `5m`.
```

Comment:
it is probably worth to mention here a shortened version [of this pr descriotion](https://github.com/zalando-incubator/postgres-operator/pull/304#issue-190550137)

Reply:
The parameter reference doesn't serve the goal of explaining underline concepts behind those parameters.